src/i18n/resources/en/common.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   clearTable: 'Clear table',
   column: 'Column',
   continue: 'Continue',
+  method: 'Method',
```

Comment:
please, ask for translations . or we just remove it from the tooltip ? we only show Linear/etc... ?

Reply:
I removed the label Method